### ISSUE TITLE
Max size of integer

### ISSUE THREAD

**Issue by kongyang80** [mswysocki/Madison-Student-Apartment-Finder#31]
Severity: Low

The max size of an integer is 2147483647 so if you enter any number larger such as 2147483648 for any integer field and add a listing, it just says 'We're sorry, but something went wrong. We've been notified about this issue and we'll take a look at it shortly.'.

This isn't really a bug. Just wanted to point it out in case this isn't the default behavior that you guys wanted to show.

**Comment by aeggum** [comment]
I wasn't able to reproduce the error message you got but I was able to create a number greater than what we are validating it to be and greater than the integer size.  

After some tinkering I found the bug and I think I at least prevented a larger than 25k number (for rent) from being used.  
Thanks for this bug, even though I haven't seen reproduced it, it made me notice a problem with our validations.
